lib/jdomodels/src/main/resources/schema/Revision-ddl.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE IF NOT EXISTS `JDOREVISION` (
   `COLUMN_MODEL_IDS` mediumblob,
   `SCOPE_IDS` blob,
   `ITEMS` JSON DEFAULT NULL,
+  `SEARCH_ENABLED` BOOLEAN DEFAULT NULL,

```

Comment:
This is not mapped automatically and we need to modify the NodeDao and the Node dto otherwise the property gets stores in the ENTITY_PROPERTY_ANNOTATIONS blob.